infra/capi/cluster-capi-infrastructure-docker/cluster-api-infrastructure-docker.yaml

```diff
@@ -389,32 +549,22 @@ spec:
     storage: true
     subresources:
       status: {}
-status:
```

Comment:
Curious as to why this section is being removed? Thanks

Reply:
Same as above